### PR TITLE
Added multiple options for custom runtimes

### PR DESCRIPTION
**Pull request by Yi-Zhou-01**
Added multiple options for custom runtimes

### CODE REVIEW

**Review by dhil — CHANGES_REQUESTED** [review]

File: core/basicsettings.ml [hunk 1]
```diff
@@ -116,11 +116,10 @@ module System = struct
               |> sync)
 
   let custom_js_runtime =
-    Settings.(option "custom_js_runtime"
+    Settings.(multi_option "custom_js_runtime"
               |> privilege `User
               |> synopsis "If link_js_runtime is set to true, then the JS compiler will link the provided file(s) rather than the standard Links JS runtime"
-              |> to_string from_string_option
-              |> convert (fun s -> Some s)
+              |> convert parse_paths
```
[the remark this answers is not in the document]
Comment (Yi-Zhou-01) [reply core/basicsettings.ml:123]: Okay, just added it :-)

**Review by dhil — APPROVED** [review]
LGTM.

**Comment by dhil** [comment]
Your patch is still failing the `rule-check`. It seems to be complaining about trailing whitespace in `bin/driver.ml`.

**Comment by Yi-Zhou-01** [comment]
> Your

**Comment by Yi-Zhou-01** [comment]
> Your patch is still failing the `rule-check`. It seems to be complaining about trailing whitespace in `bin/driver.ml`.

Oh, so the check doesn't allow any whitespaces at the end of each line?

**Comment by dhil** [comment]
> > Your patch is still failing the `rule-check`. It seems to be complaining about trailing whitespace in `bin/driver.ml`.
> 
> Oh, so the check doesn't allow any whitespaces at the end of each line?

That's right.

**Pull request merged**